status-network-contracts/src/utils/ERC20VotesUpgradeable.sol

```diff
@@ -0,0 +1,293 @@
+// SPDX-License-Identifier: MIT
+// OpenZeppelin Contracts (last updated v4.8.1) (token/ERC20/extensions/ERC20Votes.sol)
+
```

Comment:
We should add a comment here that this is the same as OZ version, except for the change that we've implemented and why.

Otherwise, another option is to rename `ERC20VotesUpgradeable` to `KarmaVotesUpgradeable` or something like that, to make it more explicit that this thing is behaving slightly differently.